### DISCUSSION TITLE
Custom USB MIDI descriptor

### DISCUSSION THREAD

**Discussion by ctag-fh-kiel** [hathach/tinyusb#1006] [Q&A]
Any advice on how to change the default USB MIDI descriptor to just having 1 midi output? (default has 1 in + 1 out).

Thx :)

**Comment by hathach** [comment]
you will to follow usb midi spec 1.0 and use the existing declaration for reference https://github.com/hathach/tinyusb/blob/master/src/device/usbd.h#L257
